plugins/ros/src/components/scenarioDrawer/components/ActionFormItem.tsx

```diff
@@ -71,7 +71,9 @@ export function ActionFormItem({
           </IconButton>
         </Box>
         <Input
-          {...register(`actions.${index}.title`)}
+          required
+          {...register(`actions.${index}.title`, { required: true })}
+          error={formState.errors?.actions?.[index]?.description !== undefined}
```

Comment:
Tittelfeltet skal vel ha feil fra `title` og ikke `description`?
```suggestion
          error={formState.errors?.actions?.[index]?.title !== undefined}
```

Reply:
nice catch